bank2ynab/__main__.py

```diff
@@ -1,22 +1,85 @@
 import logging
+import os
+import json
+from time import localtime, strftime
+from collections import namedtuple
 
+
+from b2y_utilities import get_configs, get_project_dir
 from bank_process import Bank2Ynab
-from b2y_utilities import get_configs
 from ynab_api import YNAB_API
 
 # configure our logger
 logging.basicConfig(format="%(levelname)s: %(message)s", level=logging.INFO)
 
-# Let's run this thing!
+CONFIG_FILE="account_info.json"
+
+def get_import_root():
+    return os.path.join(get_project_dir(), "imports")
+
+
+def init_import_root(budgets):
+    """
+    Create a directory structure matching the account names on ynab
+    :return:
+    """
+    root = get_import_root()
+    for budget in budgets:
+        for account in budget.accounts:
+            folder = os.path.join(root, budget.name, account.name)
+            os.makedirs(folder, exist_ok=True)
+            config = os.path.join(folder, CONFIG_FILE)
+            if not os.path.exists(config):
+                with open(config, "w") as f:
+                    json.dump({"bank_id": "DK Bankernes EDB Central"}, f)
+
+
+Job = namedtuple("Job", ("budget", "account", "filename", "bank_id"))
+
+
+def move_to_history_folder(job):
```

Comment:
I don't see the need for a history folder - is that not the function that YNAB is fulfilling? In any case, I feel like it's a separate feature.

Reply:
I found it to be a nice feature that my csvs does not disapear, upon importing and that i know when they were imported. But yes the data should be in ynab when the script is done.